protocol/json-schemas/tbdex-message-schema.json

```diff
@@ -0,0 +1,276 @@
+                        "$ref": "#/definitions/paymentInstrument"
+                    }
+                },
+                "payoutInstruments": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/paymentInstrument"
+                    }
+                }
+            },
+            "required": [
+                "description",
+                "pair",
+                "unitPrice",
+                "min",
+                "max",
+                "presentationRequestJwt",
+                "payinInstruments",
+                "payoutInstruments"
+            ]
+        },
+        "rfq": {
+            "type": "object",
+            "properties": {
+                "pair": {
+                    "type": "string"
+                },
+                "amount": {
+                    "type": "string"
+                },
+                "verifiablePresentationJwt": {
+                    "type": "string"
+                },
+                "payinInstrument": {
+                    "$ref": "#/definitions/paymentInstrument"
+                },
+                "payoutInstrument": {
+                    "$ref": "#/definitions/paymentInstrument"
+                }
+            },
+            "required": [
+                "pair",
+                "amount",
+                "verifiablePresentationJwt",
+                "payinInstrument",
+                "payoutInstrument"
+            ]
+        },
+        "quote": {
+            "type": "object",
+            "properties": {
+                "expiryTime": {
+                    "type": "number"
+                },
+                "totalFee": {
+                    "type": "string"
+                },
+                "amount": {
+                    "type": "string"
+                },
+                "paymentPresentationRequestJwt": {
+                    "type": "string"
+                },
+                "paymentInstructions": {
+                    "$ref": "#/definitions/paymentInstructions"
+                }
+            },
+            "required": [
+                "expiryTime",
+                "totalFee",
+                "amount",
+                "paymentPresentationRequestJwt",
+                "paymentInstructions"
+            ]
+        },
+        "order": {
+            "type": "object",
+            "properties": {
+                "paymentVerifiablePresentationJwt": {
+                    "type": "string"
+                },
+                "required": [
+                    "paymentVerifiablePresentationJwt"
+                ]
+            }
+        },
+        "orderStatus": {
+            "type": "object",
+            "properties": {
+                "orderStatus": {
+                    "type": {
+                        "enum": [
+                            "PENDING"
+                        ]
+                    }
+                },
+                "required": [
```

Comment:
same here, i believe required array to sit outside of properties key so it's on the same level as orderStatus